### DISCUSSION TITLE
How to prevent moving in suggestions popup while in insert mode?

### DISCUSSION THREAD

**Discussion by ghost** [helix-editor/helix#4129] [Q&A] [answered]
I prefer the beahviour of Kakoune in regards to this using `Ctrl+n` to cycle between the itens. Can I disable this behaviour?

**Comment by the-mikedavis** [comment, marked as answer]
It's not currently possible to remap keys in the completion menu.

`C-n`/`C-p` are in the keymap for the completion menu though - you should be able to use them to cycle between completion items

**Reply by ghost** [reply]
Thanks. My issue is that when I'm in INSERT mode and I navigate through the code it's often that the completion menu popup and disrupts my movements. I was looking to disabling it, but since you said that it's not possible to remap, I'll accept as an answer.
